src/react/databrowser/buckets/details/Overview.tsx

```diff
@@ -94,18 +97,37 @@ function Overview({ bucket, ingestionStates }: Props) {
   const loading = useSelector(
     (state: AppState) => state.networkActivity.counter > 0,
   );
-  const workflowsQuery = useWorkflows();
+  const workflowsQuery = useWorkflows([bucket.Name]);
   const features = useSelector((state: AppState) => state.auth.config.features);
   const { account } = useCurrentAccount();
+  const [isErrorModalOpen, setIsErrorModalOpen] = useState(false);
   useEffect(() => {
     dispatch(getBucketInfo(bucket.Name));
   }, [dispatch, bucket.Name]);
 
+  const workflows = workflowsQuery.data;
+  let attachedWorkflowsCount = 0;
+
+  if (workflows && workflows.expirations) {
+    attachedWorkflowsCount += workflows.expirations.length;
+  }
+  if (workflows && workflows.replications) {
+    attachedWorkflowsCount += workflows.replications.length;
+  }
+  if (workflows && workflows.transitions) {
+    attachedWorkflowsCount += workflows.transitions.length;
+  }
+
```

Comment:
just a suggestion to simplify the code a bit ;) 
```suggestion
  const attachedWorkflowsCount =
    (workflows?.expirations?.length || 0) +
    (workflows?.replications.length || 0) +
    (workflows?.transitions.length || 0);

```

Reply:
Yey, TS goodies come to the rescue :)